lte/gateway/c/oai/include/mme_app_ue_context.h

```diff
@@ -399,7 +399,7 @@ typedef struct ue_mm_context_s {
    */
   rau_tau_timer_t rau_tau_timer;
 
-  int nb_active_pdn_contexts;
+  unsigned int nb_active_pdn_contexts;
```

Comment:
leave this as int. We do not need the full range of unsigned int and if there is a bug (e.g., decrementing its value when it is zero), it is easier to catch.